### DISCUSSION TITLE
album primary image and jellyfin

### DISCUSSION THREAD

**Discussion by losmat** [music-assistant/support#3224] [Q&A] [answered]
Hi all,
Running into trouble with album pictures : Pictures pointing to some 404 proxyimage location. I tried to double click on that and since I'm new to music-assistant, I'm puzzled with the concept of primary image for a given album. This is especially with jellyfin (I don't have other provider yet).

I could see there is large period retention for the image in the code.
I see I also have 2 jellyfin provider when only one is running.
But result is music-assistant is not able to display a correct album picture even if there is one available in the list.

What is the best way to get these albums pictures back on the UI whitout having to go on each album image and make primary a the correct source ? 

![image](https://github.com/user-attachments/assets/eb4bc4fe-3f29-47f4-a984-a48a134324f0)

Thank you for this initiative. I was looking for a way to bridge sources and providers  for music on top of home assistant and I found the approach of music-assistant really matching what I was looking for ! Thank you again to make it real :)

**Comment by OzGav** [comment, marked as answer]
How have you ended up with two Jellyfin providers? I think you will need to remove the duplicate (if that is what it is) and then do a database reset via MA SETTINGS>>CORE>>MUSIC CONTROLLER>>ADVANCED

**Reply by losmat** [reply]
Hi, thank you for your answer/hints. I did go ahead and remove/add the Jellyfin provider. MA is looking good now.

 I don't know really how I did end up with that duplicated provider, maybe during the setup authenticating twice against it?... or an update of jellyfin, or maas or both... It's the really begining for me to use these components... probably an in between chair and keyboard issue :)
